ci-bench/src/bench_lib/benchmark.rs

```diff
@@ -0,0 +1,179 @@
+use crate::cli::Side;
+use crate::util::key::KeyType;
```

Comment:
Style nit: would be nice to split these import blocks with newlines, between `crate`, external crates and `std` imports.